core/templates/dev/head/filters/RemoveDuplicatesInArrayFilter.ts

```diff
@@ -18,6 +18,9 @@
 
 oppia.filter('removeDuplicatesInArray', [function() {
   return function(input) {
+    if (!input.filter) {
+      throw Error('Bad input: ' + JSON.stringify(input));
```

Comment:
Maybe "Bad input for removeDuplicatesInArray", so it's easy to identify just by looking at the message?

Reply:
Done